src/pygambit/game.pxi

```diff
@@ -336,6 +336,39 @@ class Game:
         g.title = title
         return g
 
+    def to_arrays(self) -> typing.List[np.array]:
+        """Reverse function to from_arrays:
+
+        To_arrays, for a given game, generates players’ payoff tables represented as numpy arrays.
+        The number of produced arrays is equal to the number of players.
+
+        Returns
+        -------
+        list of np.arrays
```

Comment:
The syntax would be `list of np.array` (the plural is implicit in numpydoc style)